### PR TITLE
Adapt cartridge-cli v2.0+

### PR DESCRIPTION
**Pull request by rosik**
In new major version of CLI the layout was changed.
This patch adds relevant comments and gitignore paths.

I didn't forget about

- [x] Tests (unnecessary)
- [x] Changelog (unnecessary)
- [x] Documentation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -76,11 +76,10 @@ Long story short, copy-paste this into the console:
 
 .. code-block:: bash
 
-    tarantoolctl rocks install cartridge-cli
-    .rocks/bin/cartridge create --name myapp
+    cartridge create --name myapp
```
Comment (rosik) [inline README.rst:79]: @lenkis could you help here, please.

It's time to update installation instructions for cli 2.0.

I guess we could move `Prerequisites` section from `Contribution` section above `Create your first application`. WDYT?

Comment (lenkis) [reply README.rst:86]: LGTM

File: README.rst [hunk 2]
```diff
@@ -65,22 +65,28 @@ Getting started
...
-Create your first application
+Prerequisites
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To get a template application that uses Tarantool Cartridge and run it,
-you need to install the `cartridge-cli` utility (supposing that
-`Tarantool <https://www.tarantool.io/en/download/>`_ is already installed).
+you need to install several packages:
+
+* ``tarantool``, ``tarantool-dev``, ``cartridge-cli``
```
Comment (lenkis) [inline README.rst:74]: let's move cartridge-cli to a standalone bullet item and provide a link to https://github.com/tarantool/cartridge-cli#installation

File: README.rst [hunk 3]
```diff
@@ -168,15 +168,15 @@ If you also need the stateful failover mode, launch an external state provider
...
 
 And set failover parameters according to ``instances.yml``. The defaults are:
 
 * State provider URI: ``localhost:4401``;
 * Password: ``qwerty``.
 
-For more details about ``cartridge-cli``, see
-https://github.com/tarantool/cartridge-cli#readme.
+For more details about ``cartridge-cli``, see its
+https://github.com/tarantool/cartridge-cli#usage.
```
Comment (lenkis) [inline README.rst:179]:
```suggestion
[usage](https://github.com/tarantool/cartridge-cli#usage).
```

File: README.rst [hunk 4]
```diff
@@ -65,22 +65,28 @@ Getting started
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To get a template application that uses Tarantool Cartridge and run it,
-you need to install the `cartridge-cli` utility (supposing that
-`Tarantool <https://www.tarantool.io/en/download/>`_ is already installed).
+you need to install several packages:
+
+* ``tarantool`` and ``tarantool-dev``
+  (see these `instructions <https://www.tarantool.io/en/download/>`_);
+* ``git``, ``gcc``, ``cmake`` and ``make``.
```
Comment (lenkis) [inline README.rst:76]:
```suggestion
* ``cartridge-cli`` (see these `<instructions https://github.com/tarantool/cartridge-cli#installation>`_)
* ``git``, ``gcc``, ``cmake`` and ``make``.
```

File: README.rst [hunk 5]
```diff
@@ -168,15 +170,15 @@ If you also need the stateful failover mode, launch an external state provider
...
 
 And set failover parameters according to ``instances.yml``. The defaults are:
 
 * State provider URI: ``localhost:4401``;
 * Password: ``qwerty``.
 
-For more details about ``cartridge-cli``, see
-https://github.com/tarantool/cartridge-cli#readme.
+For more details about ``cartridge-cli``, see its
+[usage](https://github.com/tarantool/cartridge-cli#usage).
```
Comment (lenkis) [inline README.rst:181]:
```suggestion
`usage <https://github.com/tarantool/cartridge-cli#usage>`_.
```

**Pull request merged**